src/Presentation/Console/Resources/config/services.yaml

```diff
@@ -42,7 +42,9 @@ services:
     tags: ['console.command']
     arguments:
       - '@ide.config_manager'
-
+  security_checker_command:
+    class: SecurityChecker\Command\SecurityCheckerCommand
```

Comment:
but what about task file? don't we need it?

Reply:
we don't we already have command and don't have any other entrypoints for this command